base/sparse/sparsevector.jl

```diff
@@ -30,15 +30,34 @@ end
 SparseVector(n::Integer, nzind::Vector{Ti}, nzval::Vector{Tv}) where {Tv,Ti} =
     SparseVector{Tv,Ti}(n, nzind, nzval)
 
+# Define an alias for a view of a whole column of a SparseMatrixCSC. Many methods can be written for the
+# union of such a view and a SparseVector so we define an alias for such a union as well
+const SparseVectorView{T}  = SubArray{T,1,<:SparseMatrixCSC,Tuple{Base.Slice{Base.OneTo{Int64}},Int64},false}
```

Comment:
maybe call this SparseColumnView? since it's not a view **of** a SparseVector, but rather a view of a single column of a sparse matrix **as** as a SparseVector